packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -158,6 +158,38 @@ function getCommonEventData<T extends keyof DocumentEventMap>(eventConfig: UIEve
   };
 }
 
+
+// Keep track of event handlers
+const uiEventHandlers = new Map<UIEventConfig['eventName'], {
+  captureHandler: (event: Event) => void,
+  bubbleHandler: (event: Event) => void,
+  active: boolean,
+}>;
+
+export function disableEventHandlers(eventName: UIEventConfig['eventName']): void {
```

Comment:
since you're exporting these methods, call them `{enable|disable}UIEventHandlers`